src/style.css

```diff
@@ -1 +1,105 @@
 /* add styles here */
+* {
+  box-sizing: border-box;
+  border: 0;
+
+  font-weight: 300;
+}
+
+.bar {
+  position: relative;
+  background: #ffff;
+
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  width: auto;
+  margin: 20px 0 0;
```

Comment:
Be consistent with your margins. It's recommended to add only top or bottom margin, but not both, to avoid potential margin collapse issues.